redwood-widget/src/jsMain/kotlin/app/cash/redwood/widget/HTMLElementChildren.kt

```diff
@@ -15,20 +15,33 @@
  */
 package app.cash.redwood.widget
 
+import app.cash.redwood.LayoutModifier
+import app.cash.redwood.widget.MutableListChildren.Child
 import kotlinx.dom.clear
 import org.w3c.dom.HTMLElement
 import org.w3c.dom.get
 
 public class HTMLElementChildren(
   private val parent: HTMLElement,
 ) : Widget.Children<HTMLElement> {
-  override fun insert(index: Int, widget: HTMLElement) {
+  private val _children = MutableListChildren<HTMLElement>()
+  public val children: List<Child<HTMLElement>> get() = _children
```

Comment:
Where is the public half of this used? I didn't see anything. Seems to be on all of the built-in types but I can't see a usage.

Reply:
Oh I guess it's for the parent to see the instances and their layout modifiers.

Reply:
Yep, the plan is to expose this list so the parent can access the layout modifiers using the same index for the view.